.github/workflows/test.yml

```diff
@@ -28,9 +28,9 @@ jobs:
       - name: rand_distr
         run: cargo doc --all-features --package rand_distr --no-deps
       - name: rand_chacha
-        run: cargo doc --all-features --package rand_chacha --no-deps
+        run: cargo doc --all-features --package rand_chacha --no-deps --features rand_core/getrandom
```

Comment:
Not necessary here, and the RNG crates will not normally be used alone, so I don't think there's a need.

Reply:
I just think it will be the easiest approach for us and easier to discover for users, but I am fine with leaving it as-is. Note that in this case you also need to modify accordingly the `[package.metadata.docs.rs]` section in Cargo.toml, otherwise we will get a dead link for `from_os_rng`.

Reply:
Okay, I'll take your advice. (Also because docs.rs does not support indirect dependencies.)